### PR TITLE
fix: update README.md

### PR DESCRIPTION
**Pull request by Andras-Csanyi**
## PR summary
Info about package deprecation

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,31 @@ Node.js client library to interact with various
...
+```
+npm uninstall ibm-platform-services
+npm install @ibm-cloud/platform-services
+```
+
+The following path changes also needed in the files where services are in use:
+
+```javascript
+
+// prior to package deprecation
```
Comment (padamstx) [inline README.md:60]:
```suggestion
// A 'require' statement that reflects the old package name:
```

File: README.md [hunk 2]
```diff
@@ -40,6 +40,31 @@ Node.js client library to interact with various
...
+const UserManagementV1 = require('ibm-platform-services/user-management/v1');
+
+// with the new package
```
Comment (padamstx) [inline README.md:63]:
```suggestion
// Modify this to reflect the new package name:
```

**Review by padamstx — CHANGES_REQUESTED** [review]
I suggested a few changes.  I'd also like to get @dpopp07 eyes on this PR as well.

**Review by padamstx — APPROVED** [review]
LGTM

**Review by dpopp07 — APPROVED** [review]
Looks good! 👍

**Pull request merged**